spec/controllers/robots_controller_spec.rb

```diff
@@ -4,61 +4,38 @@
 require './app/controllers/robots_controller'
 
 RSpec.describe RobotsController, type: :controller, robots: true do
+  has_a_working_api
```

Comment:
Is this referring to the `V1` api? Is it worth adding a comment saying this can be removed once the update to V2 api is complete (if that is the plan?)

Reply:
Yeah it does some very dodgy stuff to make V1 APIs be stubbed all over the place.  Removal is the plan.  This is literally everywhere in the unit tests.  It's part of my overall plan to clean up all traces of V1, but there's not much point in adding comments as I'm aware and will just be removing the comments later.

Reply:
(I've also experimented with removing them when I think the API is no longer needed in some tests and it always highlights how much more v1 is still to be removed because the tests fail)